docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md

```diff
@@ -44,4 +44,38 @@ options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_in
 dvc_client = DevCycle::DVCClient.new("dvc_server_token_hash", options, true)
```

Comment:
do we say `token_hash` or just call it `sdk_token` ?

Reply:
`sdk_token` is better. I'll make the change